ground-core/src/main/java/edu/berkeley/ground/resources/NodesResource.java

```diff
@@ -85,7 +92,9 @@ public NodeVersion createNodeVersion(@Valid NodeVersion nodeVersion, @QueryParam
     @Path("/{name}/latest")
     public List<String> getLatestVersions(@PathParam("name") String name) throws GroundException {
         LOGGER.info("Retrieving the latest version of node " + name + ".");
-        return this.nodeFactory.getLeaves(name);
+        List<String> leaves = this.nodeFactory.getLeaves(name);
+        Set<String> set = new HashSet<>(leaves); //Dedup the leaves
```

Comment:
It looks like you fixed the problem [here](https://github.com/ground-context/ground/commit/7a62579cf7bc01d91a6cc4f14347c3639a66d7f0#diff-3b873ff5189e84e28d67a01a33b18202L92). I'll check to make sure, and then I'll remove my dedup code.